src/components/TimeCardPage/TimeSheet.tsx

```diff
@@ -154,16 +155,19 @@ export default function Page() {
 
     // this hook should always run first
     useEffect(() => {
-        apiClient.getUser().then(userInfo => {
-            setUser(userInfo);
-            if (userInfo.Type === "Supervisor" || userInfo.Type === "Admin") {
-                apiClient.getAllUsers().then(users => {
-                    setAssociates(users);
-                    setSelectedUser(users[0]);
-                })
-            }
-            setSelectedUser(userInfo)
-        })
+        // Uncomment this if you want the default one loaded 
```

Comment:
If the API works, would just have the API one loaded as a default / uncommented when merging